cumulusci.yml

```diff
@@ -78,6 +78,7 @@ tasks:
                 STG_InstallScript.insertRelationshipLookups();
                 TDTM_Global_API.setTdtmConfig(TDTM_Global_API.getDefaultTdtmConfig(), 'hed');
                 STG_InstallScript.enablePreferredPhoneSync();
+                STG_InstallScript.enablePreventDeletionHierarchyCustomSettings(NULL);
```

Comment:
Not a big fan of passing "NULL" as parameters...

Reply:
@xinnanlisf For this we are utilizing same method for both new installs and upgrades so this way it is reusable, wanted to check if you have any other suggestions?

Reply:
right, by using custom setting facade. configSetting(), we can set up the default value for custom settings. This will only happen for new installs since configSetting() will only run when context.previousVersion() == null.

Reply:
Got it! thanks for the clarification. Updated based on feedback.